### DISCUSSION TITLE
Skipping orders in deliveries

### DISCUSSION THREAD

**Discussion by Matarener** [sunflower-land/sunflower-land#2603] [General]
For many novice players, the main source of sfl was the old grub shop. But now it has been replaced by a delivery system. But most new players with not the most advanced farms will get stuck on some orders that they cannot complete for various reasons at these levels. I propose to add an order skip system, similar to quests for obtaining tickets. This will improve the gameplay and gaming experience for new players and will not affect experienced players in any way.

**Comment by eliasSFL** [comment]
These hard tasks are meant to be there so that you continue to expand your land so that u can unlock buildings to make those orders

**Comment by adamhannigan** [comment]
Thanks for the feedback. This designed was intended.

Roughly every 20 orders, a player will receive an order that requires them to expand or level up. The purpose of this is to ensure players are constantly progressing their Bumpkin/Island and not just fulfilling orders for SFL.

Players can optionally complete these, but after 20 orders, 40 orders, 60 orders, 80 orders, 100 orders and then 120 orders they will have no option but to complete the difficult order.

I hope that helps! 😄

**Comment by NotKeen** [comment]
I, too, think there should be some kind of skip mechanic for deliveries. I dunno, start very conservative (1 per week or day or something). I think it's bad style to have it 100% forced and unforseen situations might arise. Maybe someone gets stuck with 100% deliveries they can't fulfill.
